### PR TITLE
SETI-376: Different version of tox python interpreters

### PR DESCRIPTION
**Pull request by janpriessnitz**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/2/ : SUCCESS in 7s
- https://ci.intgdc.com/job/zuul-tox-zuul/2/ : FAILURE in 6m 16s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/3/ : SUCCESS in 8s
- https://ci.intgdc.com/job/zuul-tox-zuul/5/ : FAILURE in 19s

**Comment by janpriessnitz** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/5/ : SUCCESS in 4s
- https://ci.intgdc.com/job/zuul-tox-zuul/7/ : FAILURE in 22s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/6/ : SUCCESS in 4s
- https://ci.intgdc.com/job/zuul-tox-zuul/8/ : FAILURE in 5m 23s

**Comment by janpriessnitz** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/7/ : SUCCESS in 5s
- https://ci.intgdc.com/job/zuul-tox-zuul/9/ : SUCCESS in 5m 17s

**Comment by janpriessnitz** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/8/ : SUCCESS in 5s
- https://ci.intgdc.com/job/zuul-tox-zuul/10/ : SUCCESS in 5m 54s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/9/ : SUCCESS in 6s
- https://ci.intgdc.com/job/zuul-tox-zuul/11/ : FAILURE in 40s

**Comment by janpriessnitz** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/10/ : SUCCESS in 6s
- https://ci.intgdc.com/job/zuul-tox-zuul/12/ : FAILURE in 35s

**Comment by janpriessnitz** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/11/ : SUCCESS in 7s
- https://ci.intgdc.com/job/zuul-tox-zuul/13/ : SUCCESS in 5m 54s

File: tox.ini [hunk 1]
```diff
@@ -28,7 +29,7 @@ commands = bindep test
 
 [testenv:pep8]
 # streamer is python3 only, so we need to run flake8 in python3
```
Comment (michalvanco) [inline tox.ini]: remove these 2 lines pls

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/15/ : SUCCESS in 39s
- https://ci.intgdc.com/job/zuul-tox-zuul/17/ : FAILURE in 5m 49s

**Comment by janpriessnitz** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/zuul-opensource-repository-check/16/ : SUCCESS in 7s
- https://ci.intgdc.com/job/zuul-tox-zuul/18/ : SUCCESS in 5m 58s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/zuul-merge-compliance-check/1/ : SUCCESS in 11s
- https://ci.intgdc.com/job/zuul-tox-zuul/19/ : SUCCESS in 5m 43s

**Pull request merged**